files/preseeds/openSUSE-Leap.autoyast

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <networking>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <partitioning  config:type="list">
+    <drive>
+      <device>/dev/sda</device>
+      <use>all</use>
+    </drive>
+  </partitioning>
+  <software>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <patterns config:type="list">
+      <pattern>base</pattern>
+      <pattern>console</pattern>
+    </patterns>
+    <packages config:type="list">
+      <package>perf</package>
+      <package>zypper-aptitude</package>
+    </packages>
+  </software>
+  <users config:type="list">
+    <user>
+      <username>root</username>
+<!-- From Leap 15.1+, password based ssh root login is disabled -->
```

Comment:
Does this mean sshd_config needs to be edited ? Logging in (as root) is used as validation after provisioning.